python/adelphi/test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+docker ~= 4.4
+tenacity ~= 7.0
+click ~= 7.1
```

Comment:
A set of dependencies which must be in place when trying to run run-adelphi-tests.py.  These were previously declared in tox.ini (minus click which is new) but now that some of that functionality has moved out to the script we have to have them available earlier.